helm/prometheus-rules/templates/loki-ruler-datasource-configmap.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.mimir.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: loki-ruler-datasource-{{ .Values.managementCluster.name }}
+  namespace: monitoring
+  labels:
+    app.giantswarm.io/kind: datasource
+data:
+  loki-ruler-{{ .Values.managementCluster.name }}.yaml: |
+    apiVersion: 1
+    datasources:
+    - access: proxy
+      editable: false
+      jsonData:
+        manageAlerts: true
+        httpHeaderName1: X-Scope-OrgID
+      secureJsonData:
+        httpHeaderValue1: {{ .Values.managementCluster.name }}
+      name: Loki-ruler-{{ .Values.managementCluster.name }}
+      type: loki
+      url: http://loki-backend.loki.svc:3100
```

Comment:
What's the difference between this datasource and the loki one?

Reply:
This one queries the backend directly, which is responsible for the rules. And with a tenant ID.
The reason for this is that when we go through the gateway, the MTP adds a list of tenants, and the ruler does not work with that.
So, as a first step, the rules will only work on the MC logs. And so does this extra datasource.

This datasource won't work for logs queries, it only works for listing rules.

I know it's not an ideal solution, but at least we can have a MVP to start with, then we can look for improvements.

Reply:
Sure that makes sense, thanks for explaining